models/dfk/dfk_hero_updates.sql

```diff
@@ -0,0 +1,95 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key='log_id',
+        incremental_strategy = 'delete+insert',
+        tags=['dfk'],
+        cluster_by=['block_timestamp']
+        ) 
+}}
+
+with
+events as (
+    select
+        *
+    from {{ ref("logs") }}
+    where {{ incremental_load_filter("block_timestamp") }}
+        and topics[0] = '0x7be34da84992130f23438d167b22f7f7a246aaaf2d7e9dd3c988ee1672fe40fc'
```

Comment:
can you add a comment here on what this topic is?

Reply:
Honestly not sure what event is this as its un-documented.
If you select distinct topics[0] from the above address, its the only topic that doesn't appear in 4byte.directory.